### PR TITLE
DPC-1388 return system information

### PR DESCRIPTION
**Pull request by MrBilnon**
### Fixes [DPC-1388](https://jira.cms.gov/browse/DPC-1388)

Create a get endpoint for v2 system and return data about the system

### Change Details

* create swagger output structs that is a subset of the data from the databases
* use those swagger output structs in the response of api

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

Screen Shot 2021-06-25 at 8 40 48 PM

### CODE REVIEW

File: ssas/systems.go [hunk 1]
```diff
@@ -82,9 +82,9 @@ type ClientToken struct {
 	gorm.Model
 	Label     string    `json:"label"`
 	Uuid      string    `json:"uuid"`
-	System    System    `gorm:"foreignkey:SystemID;association_foreignkey:ID"`
-	SystemID  uint      `json:"system_id"`
-	ExpiresAt time.Time `json:"expires_at"`
+	System    System    `gorm:"foreignkey:SystemID;association_foreignkey:ID" json:"-"`
+	SystemID  uint      `json:"-"`
+	ExpiresAt time.Time `json:"-"`
```
Comment (jonfulk) [inline ssas/systems.go:87]: we don't want to expose when it expires? Is that for security purposes? It seems like it could be helpful information.

**Review by jonfulk — APPROVED** [review]
LGTM

File: ssas/service/admin/api.go [hunk 2]
```diff
@@ -178,6 +179,52 @@ func updateGroup(w http.ResponseWriter, r *http.Request) {
...
+	systemEvent := ssas.Event{Op: "UpdateSystem", TrackingID: trackingID}
+
+	s, err := ssas.GetSystemByID(id)
+	if err != nil {
+		systemEvent.Help = fmt.Sprintf("could not find system %s", id)
+		ssas.OperationFailed(systemEvent)
+		jsonError(w, http.StatusNotFound, fmt.Sprintf("could not find system %s", id))
+		return
+	}
+
```
Comment (dhgreene) [inline ssas/service/admin/api.go:194]: We probably ought to add a verification that this system is active

File: ssas/systems.go (hunk 1 above)
Comment (dhgreene) [reply ssas/systems.go:87]: @MrBilnon care to add client token expiration date?

File: ssas/service/admin/api.go (hunk 2 above)
Comment (MrBilnon) [reply ssas/service/admin/api.go:194]: dobe

File: ssas/systems.go (hunk 1 above)
Comment (MrBilnon) [reply ssas/systems.go]: dobe

File: ssas/service/admin/api.go (hunk 2 above)
Comment (dhgreene) [reply ssas/service/admin/api.go:194]: Can we also add a unit test case?

**Review by MrMorie — DISMISSED** [review]
LGTM

**Review by dhgreene — APPROVED** [review]
💯

**Pull request merged**